Don't define getline and getdelim if they are already defined
(e.g. on Mac OS X 10.7 and later).
--- getline.c.orig	2015-12-28 00:38:39.000000000 -0600
+++ getline.c	2019-11-04 09:39:32.000000000 -0600
@@ -169,6 +169,8 @@
 #ifndef __OpenBSD__
 #ifndef __NetBSD__
 
+#ifndef HAVE_GETLINE
+
 int
 getline (lineptr, n, stream)
      char **lineptr;
@@ -178,6 +180,10 @@
   return getstr (lineptr, n, stream, '\n', 0);
 }
 
+#endif
+
+#ifndef HAVE_GETDELIM 
+
 #if defined(__FreeBSD__)
 #include <osreldate.h>
 #if __FreeBSD_version <= 800066
@@ -196,6 +202,8 @@
 }
 #endif
 
+#endif
+
 #endif /* ifndef __NetBSD__ */
 #endif /* ifndef __OpenBSD__ */
 
